### DISCUSSION TITLE
Imask Date support for day/monthName/year format in angular

### DISCUSSION THREAD

**Discussion by sonali395** [uNmAnNeR/imaskjs#731] [Ideas]
Below is an example of input field with date mask. I want to achieve masking for the format `day/monthName/year` , example : 23/Jun/1995.

Issue:
I am able to key in date in this format 23/Jan/1995 , with Jan as month name but it isn't allowing me to keyin the whole date value when I input `jan` in place of month holder which is also present in the enum provided to month block in the mask options.

**To Reproduce**
https://stackblitz.com/edit/angular-4wgdb2?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Futils.ts,src%2Fapp%2Fapp.component.ts

**Expected behavior**
It should allow me to keyin case insensitive month name values(eg: Jan, JAN, jan) in the date input. Also I want to enhance it further where I key in month number or alphabet and it displays the month name. For example, if I key in `6` then Jun should appear or I press `J` then Jan should appear

**Environment:**
 - Browser: chrome
 - "@angular/cli": "^13.2.6"
 - "angular-imask": "^6.4.2"

**Comment by uNmAnNeR** [comment]
a complex combination of `prepare` and mask config
